ishell/console.py

```diff
@@ -71,8 +71,9 @@ def walk_and_run(self, command):
 
     def print_childs_help(self):
         print("Help:")
+        pad = max([len(c) for c in self.childs.keys()])
         for command_name in sorted(self.childs.keys()):
-            print("%15s - %s" % (command_name, self.childs[command_name].help))
+            print(f"%{pad}s - %s" % (command_name, self.childs[command_name].help))
```

Comment:
very minor: in ishell/command.py the change was with old style formatting (which seems to be the ishell way) - here we are introducing a format string

Reply:
:+1: Yeah that would have been better.